test/integration/test_git_project_build.py

```diff
@@ -16,6 +16,8 @@ def git_project_path(tmp_path: Path, asset_dir: Path) -> Path:
     )
 
     subprocess.check_call(['git', 'init'], cwd=git_project_dir)
+    subprocess.check_call(['git', 'config', 'user.email', 'test@example.com'], cwd=git_project_dir)
```

Comment:
Optional:
There is another solution: you can environment variables instead of `git config` commands.
These variables are: GIT_COMMITTER_NAME, GIT_AUTHOR_EMAIL, GIT_COMMITTER_EMAIL, GIT_AUTHOR_NAME

Reply:
Or git also has a `-c` option for config parameters.